unison-cli/src/Unison/Codebase/Editor/HandleInput/Update2.hs

```diff
@@ -381,9 +381,9 @@ findCtorNames names forwardCtorNames ctorCount n =
       insertShortest m _ = m
       m = foldl' insertShortest mempty (Foldable.toList center)
       ctorCountGuess = fromMaybe (Map.size m) ctorCount
-   in if Map.size m == ctorCountGuess && all (isJust . flip Map.lookup m) [0 .. fromIntegral ctorCountGuess - 1]
+   in if Map.size m == ctorCountGuess && all (isJust . flip Map.lookup m . fromIntegral) [0 .. ctorCountGuess - 1]
```

Comment:
I think this was somehow counting up from `[0 .. MAX_INT]` instead of `[0 .. -1]` and then failing the `all`. Sneaky.